client/components/forms/counted-textarea/test/index.jsx

```diff
@@ -4,7 +4,7 @@
  */
 import { expect } from 'chai';
 import React from 'react';
-import TestUtils from 'react-addons-test-utils';
+import { createRenderer } from 'react-test-renderer/shallow';
```

Comment:
[15.6](https://github.com/facebook/react/blob/v15.6.1/docs/docs/addons-shallow-renderer.md#overview) and [current 16](https://reactjs.org/docs/shallow-renderer.html#overview) docs don't use this function, which _seems_ to be synonymous to the instantiation. How about adhering to docs as follows:

```js
import ShallowRenderer from 'react-test-renderer/shallow';
const renderer = new ShallowRenderer();
```

I don't see any mention of `createRenderer` in the docs 🤔 

This appears several times throughout the PR. I'm happy to make the change if there are no objections.